extension-reference/mobile/build-your-own-extension/events/identity/shared-state.md

```diff
@@ -2,21 +2,19 @@
 
 **MODULE\_NAME**: `com.adobe.module.identity`
 
-The Identity module shared state is created at the following times:
+The Identity extension shared state is created in the following situations:
 
 * When the module first loads and is initialized
-* After setting the user ID \(Visitor ID\)
-* When handling a sync request, regardless if the actual sync occurs.    Occurs when setting the push ID, advertising ID, or customer's custom identifiers.
+* When handling a sync request, regardless if the actual sync occurs. Occurs when setting the push ID, advertising ID, or customer's custom identifiers.
 * When a successful sync request occurs, the shared state is updated after receiving a response from the Visitor ID service.
```

Comment:
You can add "When the global privacy status changes to optedout, a shared state is set after the identifiers are cleared."